Task_DEV_5/Task_DEV_5/SecondSorter.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Text;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class SecondTextSort
+    {
+        private string path { get; set; }
+        public FileStream fstream;
+
+        public SecondTextSort(string path)
+        {
+            this.path = path;
+        }
+
+        public string ReadLine()
+        {
+            StringBuilder builder = new StringBuilder();
+            char symbol;
+            while ((symbol = (char)fstream.ReadByte()) != '\n')
+            {
+                if (fstream.Length == fstream.Position)
+                {
+                    break;
+                }
+
+                builder.Append(symbol);
+            }
+            return builder.ToString();
+        }
+
+        public void SortText()
+        {
+            using (fstream = new FileStream(path, FileMode.Open, FileAccess.ReadWrite))
+            {
+                bool sorting = true;
+                do
+                {
+                    fstream.Seek(0, SeekOrigin.Begin);
+                    string firstString = ReadLine();
+                    sorting = false;
+                    while (fstream.Length != fstream.Position)
+                    {
+                        string secondString = ReadLine();
+                        if (String.Compare(firstString, secondString) > 0)
+                        {
+                            var temp = firstString;
```

Comment:
Do we really need to do swap like this? Why can't we swap directly in the file without temp variable?

Reply:
Would be better to move swap logic to separate method.

Reply:
Done.